src/main/java/net/minestom/server/utils/NBTUtils.java

```diff
@@ -163,7 +163,7 @@ public static void loadDataIntoItem(@NotNull ItemStack item, @NotNull NBTCompoun
                 final double value = attributeNBT.getNumber("Amount").doubleValue();
                 final String slot = attributeNBT.containsKey("Slot") ? attributeNBT.getString("Slot") : "MAINHAND";
                 final String attributeName = attributeNBT.getString("AttributeName");
-                final int operation = attributeNBT.getInt("Operation").intValue();
+                final int operation = attributeNBT.getNumber("Operation").intValue();
```

Comment:
You can just use NBTCompound#getAsInt

See: https://github.com/jglrxavpok/Hephaistos/blob/master/src/main/kotlin/org/jglrxavpok/hephaistos/nbt/NBTCompound.kt#L223

Reply:
No, its throw a error if Operation is set to byte

Reply:
Are you talking about `getInt`? Because I am talking about `get*As*Int`, which should have the exact same behaviour as your code.